packages/ui/src/BreadCrumbs.tsx

```diff
@@ -3,21 +3,31 @@
 import { clsx } from 'clsx'
 import { usePathname } from 'next/navigation'
 import { Icon } from '@filecoin-foundation/ui/Icon'
-import { InternalTextLink } from '@filecoin-foundation/ui/TextLink'
 import { capitalize, truncate } from '@filecoin-foundation/utils/stringUtils'
 import { CaretRight } from '@phosphor-icons/react/dist/ssr'
+import Link from 'next/link'
 import type { Route } from 'next'
 
 const HOME_PATH = '/'
 const HOME_LABEL = 'Home'
 
-export function BreadCrumbs() {
+const gapMap = {
+  small: 'gap-1.5',
+  default: 'gap-2',
+  large: 'gap-2.5',
```

Comment:
@CharlyMartin - I don't actually understand this. 🙈 We don't seem to be using it?